routers/api/v1/repo/file.go

```diff
@@ -597,3 +599,88 @@ func GetContentsList(ctx *context.APIContext) {
 	// same as GetContents(), this function is here because swagger fails if path is empty in GetContents() interface
 	GetContents(ctx)
 }
+
+// GetFileHistory get a file's commit history
+func GetFileHistory(ctx *context.APIContext) {
+	// swagger:operation GET /repos/{owner}/{repo}/git/history/{filepath} repository repoGetFileHistory
+	// ---
+	// summary: Get the commit history of a file or directory
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: owner
+	//   in: path
+	//   description: owner of the repo
+	//   type: string
+	//   required: true
+	// - name: repo
+	//   in: path
+	//   description: name of the repo
+	//   type: string
+	//   required: true
+	// - name: filepath
+	//   in: path
+	//   description: filepath of the file/directory
+	//   type: string
+	//   required: true
+	// - name: ref
+	//   in: query
+	//   description: "The name of the branch. Default the repository’s default branch (usually master)"
```

Comment:
```suggestion
	//   description: "The name of the ref (branch/tag). Default the repository’s default branch"
```

The default branch name is `main` in modern days.

Reply:
OK, this was just copied from another api desc (don't know which is was, but will update this)

Reply:
https://github.com/go-gitea/gitea/pull/17652/commits/c690801cea1c52e06e73b37a6c89ca9a9234c2f8

Reply:
Well, I hope I understood correctly .... 😂

Maybe other developers can help to double confirm.